Source/OxyPlot/Rendering/OxySize.cs

```diff
@@ -31,21 +41,33 @@ public struct OxySize : IFormattable
         public OxySize(double width, double height)
             : this()
```

Comment:
I think calling a default constructor on a struct is not needed.
All memory is zeroed by default.